lib/src/main/java/org/team100/lib/hid/OperatorControl.java

```diff
@@ -26,6 +26,10 @@ default boolean pivotToAmpPosition(){
         return false;
     }
 
+    default boolean driveToNote() {
```

Comment:
why do you want both the driver and the operator to be able to trigger the drive-to-note behavior?